ext/RMagick/extconf.rb

```diff
@@ -67,8 +67,7 @@ def configure_compile_options
       if RUBY_PLATFORM !~ /mswin|mingw/
 
         # Check for compiler. Extract first word so ENV['CC'] can be a program name with arguments.
-        config = defined?(RbConfig) ? ::RbConfig : ::Config
```

Comment:
I guess this was some ancient Ruby thing?

Reply:
Yes :)